network/dhcpd.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2015 Vyronas Tsingaras <vtsingaras@it.auth.gr, Aristotle University of Thessaloniki
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: dhcpd
+short_description: Simple module that sets/reads entries from an ISC-DHCP server
+description:
+    - Module that interfaces with an ISC-DHCP server using OMAPI to add/remove host objects or lookup lease objects
+version_added: "0.0"
+author: Vyronas Tsingaras
+requirements:
+    - This module requires the Python pypureomapi module
+    - You should have configured your dhcp server to enable OMAPI
+options:
+    option_name:
```

Comment:
this is a error in document format, option_name is only a place holder for the real option name, e.g. state. So should remove it and fix the indentation.